.travis.yml

```diff
@@ -4,6 +4,20 @@ go: 1.14.x
 os: linux
 dist: bionic
 
+# Don't indent any line more than one level to avoid '\n' characters in the converter string,
+# which breaks the regex free-spacing modifier. Also, do NOT include comments in the regex string.
+if: >-
+  ((branch = master) AND (tag IS NOT present))
+  OR (
+  (repo = "kubernetes-sigs/minibroker")
+  AND (tag IS present)
+  AND (branch ~= /^(?x)
+  v(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)
+  (?:-((?:0|[1-9][0-9]*|[0-9]*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9][0-9]*|[0-9]*[a-zA-Z-][0-9a-zA-Z-]*))*))?
+  (?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?
+  $/)
```

Comment:
From before, this is (assumed to be) semver.

Reply:
Included in the comment in https://github.com/kubernetes-sigs/minibroker/pull/91/commits/10a397891c1895f064dd97a5c42e241b77e197e4.